### PR TITLE
Add the update support with the least possibly modification

### PR DESCRIPTION
**Pull request by shroffk**
Adding the minimum update feature
- [ ] @arkilic 
- [ ] @danielballan 

I intend to do a (much needed) cleanup/rewrite of a significant portion of this library and will try to address the rest of the feature requests then.

### CODE REVIEW

**Comment by ghost** [comment]
:+1: Let me know what you need for the rewrite, I can give you a hand. We should have done this 3 years ago when we first discussed it

**Comment by danielballan** [comment]
Thanks, @shroffk. Looks good to me. I'll let @arkilic review and merge.

File: test/OlogClientTest.py [hunk 1]
```diff
@@ -7,22 +7,29 @@
...
+        return None
+        
 class TestCreateClient(unittest.TestCase):
     
     def testCreateClient(self):
         '''
         Simple test to create a ologClient
         '''
-        client = OlogClient(url='http://localhost:8080/Olog')
+        client = OlogClient(url=getDefaultTestConfig('url'))
```
Comment (ghost) [inline test/OlogClientTest.py:30]: Much nicer!

File: test/_testConf.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+example file
+cat ~/pyOlog.conf
+[DEFAULT]
+url=http://localhost:8000/Olog
+"""
+
+def __loadConfig():
+    import os.path
+    import ConfigParser
+    dflt={'url':'https://130.199.219.79:9992/Olog'}
```
Comment (ghost) [inline test/_testConf.py]: Now we know your IP address :)

File: test/OlogClientTest.py (hunk 1 above)
Comment (shroffk) [inline test/OlogClientTest.py:30]: I look at this library and compared to the java one it needs significant work, this is just one example of cleanup

Comment (ghost) [inline test/OlogClientTest.py:30]: I have quite a few ideas. Let's discuss tmrw or sometime next week when you have time.

File: test/_testConf.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
 def __loadConfig():
     import os.path
     import ConfigParser
-    dflt={'url':'https://130.199.219.79:9992/Olog'}
+    dflt={'url':'https://localhost:8181/Olog'}
```
Comment (ghost) [inline test/_testConf.py:14]: :hammer:

**Comment by danielballan** [comment]
@shroffk We currently use SimpleOlogClient. This change was made to OlogClient. Would adding this to SimpleOlogClient be as simple as copying the method updateLog over?

**Comment by danielballan** [comment]
Actually I think I figured it out for myself. I should have spend more time just reading the code. :- D

**Comment by shroffk** [comment]
I am glad you figured it ouw, I was just looking into it...i think the simpleOlogClinet might not be using the Log objects...
I apologize you had to go thorugh the code...this was my first every python project and some of the code reads like java :)

**Pull request merged**